src/business/usecase/episodes/getAllEpisodesBySeriesId.ts

```diff
@@ -0,0 +1,44 @@
+import { Episode } from "../../entities/episode";
+import { EpisodesGateway } from "../../gateways/episodesGateway";
+
+export class GetAllEpisodesBySeriesIdUC {
+  constructor(private episodesGateway: EpisodesGateway) { }
+
+  public async execute(
+    input: GetAllEpisodesBySeriesIdInput
+  ): Promise<GetAllEpisodesBySeriesIdOutput> {
+    const episodes = await this.episodesGateway.getAllEpisodesBySeriesId(
+      input.series_id
+    );
+    return {
+      episodes: episodes.map(
+        (episode: {
+          getId: () => any;
+          getTitle: () => any;
+          getPicture: () => any;
+          getSynopsis: () => any;
+        }) => ({
+          id: episode.getId(),
+          title: episode.getTitle(),
+          picture: episode.getPicture(),
+          synopsis: episode.getSynopsis()
+        })
+      )
+    };
```

Comment:
Isso ficou muito estranho. A função `this.episodesGateway.getAllEpisodesBySeriesId` retorna um array de objetos da classe `Episode`. 
Ficaria muito mais claro assim:

```
return {
	episodes: episodes.map((episode: Episode) => {
		return {
	      id: episode.getId(),
          title: episode.getTitle(),
          picture: episode.getPicture(),
          synopsis: episode.getSynopsis()
		}
	})
}
```